Deployment Guides/Provisioning-Islandora-on-Ubuntu-16.md

```diff
@@ -0,0 +1,1788 @@
+         SSLCertificateKeyFile /etc/ssl/private/ssl-cert-snakeoil.key
+
+         <Directory />
+            Options FollowSymLinks
+            AllowOverride None
+         </Directory>
+         <Directory /var/www/drupal7/>
+            Options Indexes FollowSymLinks MultiViews
+            AllowOverride all
+            Order allow,deny
+            allow from all
+         </Directory>
+
+         # Setup for proxy to image servers
+         AllowEncodedSlashes On
+         ProxyRequests Off
+         ProxyPreserveHost On
+         <Proxy *>
+             Order deny,allow
+             Allow from all
+         </Proxy>
+
+         # These are for Adore-Djatoka, and work fine over http but not over https
+	 # - see https://groups.google.com/forum/?pli=1#!msg/islandora/yWlpQtL8gEU/cxF4ZmvbY8YJ
+         # - this port is correct if Adore-Djatoka is installed under the same Tomcat as Fedora. Change if installed with Blazegraph.
+         ProxyPass /adore-djatoka http://localhost:8080/adore-djatoka
+         ProxyPassReverse /adore-djatoka http://localhost:8080/adore-djatoka
+
+         # These are for Cantaloupe. Don't expose backend's tech/ver/brand. They work fine over both http and https
+         # - This port is correct if Cantaloupe is installed under the same Tomcat as Fedora. Change if using standalone Cantaloupe.
+         RequestHeader set X-Forwarded-Path /
+         ProxyPass /iiif/2 http://127.0.0.1:8080/iiif/2 nocanon
+         ProxyPassReverse /iiif/2 http://127.0.0.1:8080/iiif/22
+
+         # Some SSL tweaks
+         ServerName <your-server-FQDN>
+         SSLProtocol all -SSLv2 -SSLv3
+         SSLCipherSuite HIGH:MEDIUM:!aNULL:!MD5:!SEED:!IDEA:!RC4
+         SSLHonorCipherOrder on
+     </VirtualHost>
+</IfModule>
+```
+Note: You will need to modify the port specifications in the `ProxyPass` clauses above depending on where you install Adore-Djatoka and Cantaloupe. See below.
+
+Note: the PHP module `uploadprogress` no longer appears to work. The `pecl` installation documented in "Provisioning Islandora on Ubuntu" failed to compile, and the installation below, from DigiBESS, still throws errors in the logs. Enable the apache ssl module if you are using https. Update the memory, post and upload file size limits only if you are sure you have the available memory, or when preparing to go live with a production site.
+```
+a2enmod rewrite  
+
+a2enmod proxy
+
+a2enmod proxy_http  
+
+a2enmod headers
+
+# - do these if you are enabling https
+#a2enmod ssl
+
+#a2ensite default-ssl
+
+apt-get install php-uploadprogress
+
+sed -i '929iextension=uploadprogress.so' /etc/php/7.1/apache2/php.ini 
+
+#sed -i "s|memory_limit = 128M|memory_limit = 512M|g" /etc/php/7.1/apache2/php.ini
+
+#sed -i "s|post_max_size = 8M|post_max_size = 2048M|g" /etc/php/7.1/apache2/php.ini
+
+#sed -i "s|upload_max_filesize = 2M|upload_max_filesize = 2048M|g" /etc/php/7.1/apache2/php.ini
+
+echo "apc.shm_size = 64M" >> $APC_CONFIG_FILE
```

Comment:
APC even used anymore?

Reply:
Probably not, but I'm not an expert, so I leave that to you.

Reply:
It is not used anymore.  

Probably should use opcache instead but not sure about including that here though since the settings could impact people who are actively changing their code semi-frequently. Curious what kind of luck others have had with it.

Reply:
There's a note in the 'install.properties' section that mentions that APC is deprecated in favour of Zend Opcache, which is probably based on what I found when trying to install and configure APC. The note says "to be dealt with at a later date". I'm adding another comment at this point.